content/influxdb/v2.1/notebooks/create-notebook.md

```diff
@@ -9,101 +9,157 @@ menu:
-- To export to the dashboard, click **Export to Dashboard**.  
+- To export to the dashboard, click **Export to Dashboard**.
+
+## Add a data source cell
+
+Add a [data source cell](/influxdb/v2.1/notebooks/overview/#data-source) to pull information into your notebook.
+
+To add a data source cell, do the following:
+1. Click the {{< icon "notebook-add-cell" >}}.
+2. Select **Flux Script** or **Query Builder** as your input, and then select or enter the bucket to pull data from.
+3. Select filters to narrow your data.
+4. Select **Preview** (**CTRL + Enter**) or **Run** in the upper left dropdown menu.
+
+## Add a validation cell
+
+A validation cell uses the **Table** [visualization type](/influxdb/v2.1/visualize-data/visualization-types/) to display query results from a data source cell.
+
+To add a **Table** visualization cell, do the following:
 
-### Add an action cell
+1. Click the {{< icon "notebook-add-cell" >}}.
+2. Under **Visualization**, click **Table**.
+
+## Add a visualization cell
+
+Add a visualization cell to render query results as a [Visualization type](/influxdb/v2.1/visualize-data/visualization-types/).
+
+To add a Table visualization cell, do the following:
+
+1. Click the {{< icon "notebook-add-cell" >}}.
+2. Under **Visualization**, select one of the following visualization cell-types:
+
+   - Table
+   - Graph
+   - Note
+
+To modify a visualization cell, see [use visualization cells](#use-visualization-cells).
+For detail on available visualization types and how to use them, see [Visualization types](/influxdb/v2.1/visualize-data/visualization-types/).
+
+## Add an action cell
+
+Add an [action cell](/influxdb/v2.1/notebooks/overview/#action) to filter and apply changes to your data, for example, create an [alert](/influxdb/v2.1/monitor-alert/), process data with a [task](/influxdb/v2.1/process-data/manage-tasks/), or output data to a bucket.
+
+{{% warn %}}
+If your cell contains a custom script that uses any output function to write data to InfluxDB (for example: the `to()` function) or sends data to a third-party service, clicking Preview will write data.
+{{% /warn %}}
 
-Select one of the following action cell-types:
+- [Add an Alert cell](#add-an-alert-cell)
+- [Add an Output to Bucket cell](#add-an-output-to-bucket-cell)
+- [Add a Task cell](#add-a-task-cell)
 
-- To recieve a notification, select [Alert](#alert) and complete the steps below.
-- To write output to a bucket, select [Output to Bucket](#output-to-bucket) and complete the steps below.
-- To create a task, select [Task](#task) and complete the steps below.
+### Add an Alert cell
 
-#### Alert
+To add an [alert](/influxdb/v2.1/monitor-alert/) to your notebook, do the following:
 
 1. Enter a time range to automatically check the data and enter your query offset.
 2. Customize the conditions to send an alert.
 3. Select an endpoint to receive an alert:
    - Slack and a Slack Channel
    - HTTP post
    - Pager Duty
```

Comment:
```suggestion
   - PagerDuty
```

Reply:
I submitted a fix for this to the UI repo (https://github.com/influxdata/ui/pull/3335). "PagerDuty" is the registered trademark name.

Reply:
This change just got merged into the UI repo, so we're probably good to make it here and future-proof it.